src/app/page/Home/Home.js

```diff
@@ -382,7 +382,11 @@ const SignInHeaderComponent = ({ history }) => {
  */
 const HomePage = ({ auth, history, profile }) => {
   const org = useOrganization();
+
   useFirestoreConnect(() => {
+    if (!auth.uid) {
+      return [];
+    }
```

Comment:
Don't connect if no auth object is provided?

Reply:
yes

Reply:
I mean, its a hook, so we cant have an if outside of it

Reply:
:+1: